geode-for-redis/src/integrationTest/java/org/apache/geode/redis/internal/commands/executor/set/AbstractSDiffIntegrationTest.java

```diff
@@ -55,12 +59,34 @@ public void sdiffErrors_givenTooFewArguments() {
   }
 
   @Test
-  public void sdiffstoreErrors_givenTooFewArguments() {
-    assertAtLeastNArgs(jedis, Protocol.Command.SDIFFSTORE, 2);
+  public void sdiff_returnsAllValuesInSet() {
+    String[] values = createKeyValuesSet();
+    assertThat(jedis.sdiff("{user1}setkey")).containsExactlyInAnyOrder(values);
```

Comment:
Since this "{user1}setkey" String is used in multiple places, it might be good to make it a constant in the class.